### PR TITLE
Remove scaling parameter from configs

### LINKED ISSUE #29: Incorrect description of acoustic or diffusive scaling in examples and tutorials
Diffusive scaling was removed from Musubi because of the multilevel simulations.
For the simulations of incompressible flows, however, one uses diffusive scaling where dt quarters when dx is halved - keeping Omega fixed. Mesh convergence studies are also performed following this.

In the current version of Musubi, Omega is internally calculated from dt and it is a users responsibility to set dx, dt, etc accordingly so that a correct Omega is maintained for diffusive scaling, if performing mesh convergence studies.

Many tutorial examples are however confusing because they have if/then statements for the user to chose acoustic or diffusive scaling. All those testcases will need to be accordingly modified and documented.

### PR DESCRIPTION
**Pull request by haraldkl**
Fixes #29

The diffusive scaling does not work properly for multilevel.
Hence users do not really have a choice, and we should remove
the option to provide a scaling strategy, as it otherwise is
confusing.

### CODE REVIEW

**Comment by jainka** [comment]
removing only the scaling parameter from configs will not be enough. 
There are several **if diffusive scaling** like statements in the config files that will need to be corrected.

**Comment by haraldkl** [comment]
@jainka I know. This is merely the start of this work to address #29 and discuss the changes. Actually, as it stands this deactivates passive-scalar, for example, so we need to also do more code changes.

**Comment by haraldkl** [comment]
@KannanMasilamani Please have a look at the changes so far. I don't think the stuff in `doc_pages/tutorials` is still used, but the rest should show up in the documentation. After this the request get's larger with all the various examples needing the scaling parameter removed.

**Comment by jainka** [comment]
it seems that this will require a re-discussion. This appears like a lot of work and I am not sure if it is necessary. Perhaps the easier would be to reinstate diffusive scaling and send an error to the user if diffusive is used in case of multilevels? 
Something like **if maxlevel-minlevel>0 then write(logUnit(1),*)'ERROR: Choose scaling = "acoustic" for**

The way it stands currently is that there are acoustic and diffusive differentiations at many places, see for example mus_scheme_header_module. In fact in the scheme header module there is a good differentiation whereby the user is requested to use acoustic or diffusive depending on scheme kind. 

Lets re-discuss this as to me reinstatement of diffusive sounds like a more reasonable and much easier option.

File: doc_pages/tutorial/FlowAroundCylinder/steady.md [hunk 1]
```diff
@@ -151,7 +135,6 @@ if acoustic_scaling == true then
   u_mean_L  = 2.0*u_in_L/3.0
   nu_L      = nu_phy*dt/dx^2.
   omega     = 1.0/(3.0*nu_L+0.5)
```
[the remark this answers is not in the document]
Comment (haraldkl) [reply doc_pages/tutorial/FlowAroundCylinder/steady.md]: Changed in 6bf41ea69cb31aa2526e010bbd377991a6f045aa

File: doc_pages/tutorial/FlowAroundCylinder/steady.md [hunk 2]
```diff
@@ -169,11 +152,12 @@ else
...
   u_in_L    = u_in_phy*dt/dx
   u_mean_L  = 2.0*u_in_L/3.0
-  scaling   = 'diffusive'
 
 end
 ```
 
+Note that the actual relevant setting for Musubi here is only the `dt`.
+
 > To get better results, we could also change the Mach number which is a
```
[the remark this answers is not in the document]
Comment (haraldkl) [reply doc_pages/tutorial/FlowAroundCylinder/steady.md]: Changed in 6bf41ea69cb31aa2526e010bbd377991a6f045aa

**Comment by haraldkl** [comment]
OK, thanks @jainka, @KannanMasilamani with the conclusion to keep reading the scaling parameter, this PR is now ready to be merged, I think. We will abort if there is a multi-level mesh with diffusive scaling requested, or if the scheme requires diffusive scaling and we are on the multi-level mesh (though acoustic scaling is requested).

**Pull request merged**